Expensify/expensify-common
# Allow communication with google from our secure iframes for recaptcha

@tgolen, will you please review this?

Event communication between our secure iframe and google are currently restricted. While it looks like this is not fully necessary to allow recaptcha to function properly, it throws a lot of nasty JS errors (this is happening with the old and new import dialogs)

Fixes https://github.com/Expensify/Expensify/issues/56588

# Tests/QA
1. Import the `expensidemo` credit card with the console open
    - **Verify** there are no JS errors
